providers/terraform-provider-csbsqlserver/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/cloudfoundry/csb-brokerpak-aws/terraform-provider-csbsqlserver
+
+go 1.20
+
+require (
+	github.com/denisenkom/go-mssqldb v0.12.3
```

Comment:
Do we want to favour Microsoft's fork of this module?
https://github.com/microsoft/go-mssqldb

It seems to be currently more active and I would expect it to have more strict collaboration guidelines and better security audits (although that's not necessarily true).